<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QxrdTestScanGeneratorWindow</class>
 <widget class="QMainWindow" name="QxrdTestScanGeneratorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="m_GenerateStopButton">
          <property name="text">
           <string>Stop</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_GenerateOnceButton">
          <property name="text">
           <string>Once</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_Generate10HzButton">
          <property name="text">
           <string>10Hz</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_Generate1SecButton">
          <property name="text">
           <string>1Sec</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_Generate10SecButton">
          <property name="text">
           <string>10Sec</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_NewQWTButton">
          <property name="text">
           <string>New QWT</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_NewDataVisButton">
          <property name="text">
           <string>New DataVis</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>358</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>758</width>
          <height>458</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuGenerate">
    <property name="title">
     <string>Generate</string>
    </property>
    <addaction name="m_ActionStopGeneration"/>
    <addaction name="m_ActionGenerateOnce"/>
    <addaction name="m_ActionGenerate10Hz"/>
    <addaction name="m_ActionGenerate1Sec"/>
    <addaction name="m_ActionGenerate10Sec"/>
   </widget>
   <widget class="QMenu" name="menuWindows">
    <property name="title">
     <string>Windows</string>
    </property>
    <addaction name="m_ActionNewQWTWindow"/>
    <addaction name="m_ActionNewDataVisWindow"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuGenerate"/>
   <addaction name="menuWindows"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="m_ActionStopGeneration">
   <property name="text">
    <string>Stop Generation</string>
   </property>
  </action>
  <action name="m_ActionGenerateOnce">
   <property name="text">
    <string>Generate Once</string>
   </property>
  </action>
  <action name="m_ActionGenerate10Hz">
   <property name="text">
    <string>Generate at 10Hz</string>
   </property>
  </action>
  <action name="m_ActionGenerate1Sec">
   <property name="text">
    <string>Generate every second</string>
   </property>
  </action>
  <action name="m_ActionGenerate10Sec">
   <property name="text">
    <string>Generate every 10 Seconds</string>
   </property>
  </action>
  <action name="m_ActionNewQWTWindow">
   <property name="text">
    <string>New QWT Window</string>
   </property>
  </action>
  <action name="m_ActionNewDataVisWindow">
   <property name="text">
    <string>New Data Vis Window</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
